CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change log
 
+## [v0.7.2] - unreleased
```

Comment:
Please remove version and keep the 'unreleased' only.

Reply:
Done